src/libs/tools/tests/testtool_backendbuilder.cpp

```diff
@@ -546,3 +548,92 @@ TEST (BackendBuilder, resolveDoubleRecommends)
 	EXPECT_EQ (bb.cbegin ()[1], PluginSpec ("a"));
 	EXPECT_EQ (bb.cbegin ()[2], PluginSpec ("c"));
 }
+
+static int checkconfLookup_a_abc (ckdb::Key * errorKey, ckdb::KeySet * config)
+{
+	ckdb::Key * k = ckdb::ksLookupByName (config, "/a", 0);
+	if (k && strcmp (ckdb::keyString (k), "abc") == 0)
```

Comment:
I mean: the mocks could simply append something and return 1, or return 0/-1; without any `if`.